tests/components/ovo_energy/test_config_flow.py

```diff
@@ -5,15 +5,19 @@
 import aiohttp
 
 from homeassistant import config_entries
-from homeassistant.components.ovo_energy.const import DOMAIN
+from homeassistant.components.ovo_energy.const import CONF_ACCOUNT, DOMAIN
 from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
 from homeassistant.core import HomeAssistant
 from homeassistant.data_entry_flow import FlowResultType
 
 from tests.common import MockConfigEntry
 
 FIXTURE_REAUTH_INPUT = {CONF_PASSWORD: "something1"}
-FIXTURE_USER_INPUT = {CONF_USERNAME: "example@example.com", CONF_PASSWORD: "something"}
+FIXTURE_USER_INPUT = {
+    CONF_USERNAME: "example@example.com",
+    CONF_PASSWORD: "something",
+    CONF_ACCOUNT: "123456",
```

Comment:
It looks like account is no longer used, but it's also added to this fixture?

Reply:
Account is used, just under the `custom_account_id` property in the client which overrides the default in requests